modules/ROOT/pages/create-tokenization-service.adoc

```diff
@@ -0,0 +1,37 @@
+* A tokenization format, which describes the format and how data is tokenized. See xref:tokenization-formats.adoc[Tokenization Formats].
+
+== Create the Tokenization Service
+
+. Go to the *Runtime Manager­>Tokenization Service* page to create the service.
+. Click *Create Tokenization Service*. The prerequisites you completed provide the information you need to fill out the form. This example shows the form filled in with the following values: +
+  * In the *Runtime fabric* drop-down list, select the Runtime Fabric to deploy the tokenization service to.
+  * Select the tokenization format from the *Formats* drop-down list. You can assign more than one, or all, formats to one tokenization service.
+  * Select the secret group from the *Secret Group* drop-down list.
+  * Select the number of tokenization service replicas to run. The tokenization service runs on worker nodes in Runtime Fabric.
+  * Optionally, you can adjust the logging level for the tokenization service.
+. Click *Build and Deploy* to create the tokenization table. +
+A mapping table, which contains a large table of randomizations that are used at the core of tokenizing and detokenizing pre-builds. This is not a one-to-one table of mappings--it is used during internal steps to swap in and swap out randomizations in place of the actual data. The actual algorithm consists of these three steps:
+  ** First stage transformation: Perform AES­FFX[radix] specific domain encryption using first stage symmetric key and tweak. This will redistribute the information contained within the data to be tokenized across all characters of the extracted data.
+  ** Second stage transformation: Look­up and replace chunks of the clear­text data with the appropriate precomputed randomization in the mapping table.
+  ** Third stage transformation: Perform AES­FFX[radix] specific domain encryption using third stage symmetric key and tweak. This will redistribute the information mapped by the multiple table look­ups of the preceding step across all of the extracted.
+
+The tokenization mapping table build is a one-time action and can take awhile to build, depending on the size of the tokenization format. For example, an SSN-only table less than 200 MB in size can build in two minutes, but a larger format such as “lax alphanumeric” might take up to 20 minutes to build.
+
+If many, or all, of the formats are selected, it takes much longer, as a table with all formats is approximately 2 GB in size.
+
+== API Gateway
+
+You must have an API Gateway to protect the tokenization service. An API Gateway, proxies requests to the tokenization service, provides authorization and authentication, and provides JSON protection for the tokenization service.
+
+In API Manager, xref:2.x@api-manager::getting-started-proxy.adoc[set up the API gateway Mule app] for the Mule app to serve as the proxy.
```

Comment:
So I'm confused. Is an API gateway in the form of a Mule app? Or are we talking about a gateway to a Mule app (that might be any app)? I thought the tok. service was the proxy? Can we have an image showing all the players, named consistently, in their relative relationships?  Or am I reading this in a different order than the customer would?